app/assets/stylesheets/widgets/home_page/blog_stories.scss

```diff
@@ -242,7 +242,7 @@
         display: inline-block;
         padding: 5px 10px 5px 20px;
         /*background:#f7f6f0;*/
-        /*border-left:5px solid #a82439;*/
+        /*border-left:5px solid #98002e;;*/
```

Comment:
Yes. If there's a reason for not removing them, they should include a comment explaining why. Otherwise they just tend to cause confusion.